### ISSUE TITLE
[Client] - How to install on iPhone

### ISSUE THREAD

**Issue by sfspaulding** [sgtaziz/WebMessage#182]
Hi. I was able to get this up on running on my PC (thank you Ben Foltz). I have Cydia (via unc0ver) installed on my iPhone XR that was running iOS 14.4.1 (I JB'ed it yesterday). As far as I can tell, the relevant code is over here at this accompanying resource: https://github.com/sgtaziz/WebMessage-Tweak

Trying to figure out how to install the accompanying code/software/app on my iPhone. I've never done this before, so an ELI5 explanation would be helpful. Thanks in advance.

**Comment by ZXSwire3** [comment]
hey, you just need to add the repo to cydia on your phone the repo is 

https://sgtaziz.github.io/repo

then search and install webmessage from cydia, afterwards you just need to go to your settings and scroll down to the webmessage section and toggle any options that you would like to use and then find the IP of your device which should be shown in the webmessage settings. Just take the IP of the device and then put that into the client on your desktop and it should just connect.

**Comment by matskobinks** [comment]
> 

Im using a iPhone XR, iOS 14.2 with Taurine jailbreak.
I've been trying with unc0ver, same result.

**Comment by ZXSwire3** [comment]
Have you tried reinstalling webmessage on your device? there is this other thread that was similar that might help you with your issue, but I was also the only person that responded, but I'm not sure what else to do besides just resetting the device, but honestly I don't think that a tweak is really worth that. The only reason that I did it was because I had an old iphone that I use just for this tweak. https://github.com/sgtaziz/WebMessage/issues/183
